_posts/2020-11-05-markdown-tutorial.markdown

```diff
@@ -0,0 +1,133 @@
+
+## Basic Markdown Syntaxes
+Syntaxes are set of rules for grammar and spelling that defines the combination of symbols considered to be correctly structured in a particular programming language.
+
+##### Headings
+Headings in Markdown are any line which is prefixed with the hashtag (#) symbol. The number of hashtags represents the heading level. A total of 6 levels can be made with 6 hashtags but in most writings, only 3 is commonly used. The syntax in block quotes are showing what the text looks like in Markdown.
+
+```
+# Heading 1
+## Heading 2  
+### Heading 3
+```
+# Heading 1
+## Heading 2  
+### Heading 3
+
+##### Texts
+When writing text in Markdown, you can place emphasis on a word by wrapping it in asterisks (*) or underscores (_). They can range from 1 to 3 symbols based on the extent of the emphasis. The syntax in block quotes are showing what the text looks like in Markdown.
+```
+*Item 1*  
+**Item 2**  
+__Item 3__
+```
+*Item 1*  
+**Item 2**  
+__Item 3__
+
+When adding a link to a text, the text will we wrapped with the links in square brackets [] followed by the URL to be linked in brackets (). The syntax in block quote is showing what the text looks like in Markdown.
+```
+[Atom](https://atom.io/)
+```
+[Atom](https://atom.io/)
+
+When writing quotes, the text should start with the greater than sign (>) that converts the text to a block text. The syntax in block quote is showing what the text looks like in Markdown.
+```
+> On the journey to becoming you must persevere.
+```
+> On the journey to becoming you must persevere.
+
+##### Images
+Images are written in similar format as when attaching a link to a text. The only difference is that the line starts with the exclamation mark (!). The syntax in block quote is showing what the text looks like in Markdown.
+```
+![octocat](https://github-atom-io-herokuapp-com.freetls.fastly.net/assets/index-octonaut-8ece2623b8966578e2414c89b7b7190cb56339d1f8b7d260adf62110ce9f39c4.svg)
+```
+![octocat](https://github-atom-io-herokuapp-com.freetls.fastly.net/assets/index-octonaut-8ece2623b8966578e2414c89b7b7190cb56339d1f8b7d260adf62110ce9f39c4.svg)
+
+##### Lists
+Lists are written with the following symbols (*, - or +) or numbers at the beginning of the line. For more formatting, the line can also be indented to create nested lists. The lists can be in ordered or unordered forms. The syntax in block quotes are showing what the text looks like in Markdown.
+```
+Ordered
+1. Item 1
+1. Item 2
+1. Item 3
+   1. Item 3a
+   1. Item 3b
+
+Unordered
+* Item 1
+* Item 2
+  * Item 2a
+  * Item 2b
+```
+Ordered
+1. Item 1
+1. Item 2
+1. Item 3
+   1. Item 3a
+   1. Item 3b
+
+Unordered
+* Item 1
+* Item 2
+  * Item 2a
+  * Item 2b
+
+##### Horizontal lines
+Horizontal lines (---) are used to denote visual separation between different sections of a text. The syntax in block quote is showing what the text looks like in Markdown.
+```
+---
```

Comment:
Maybe put some some text that is being separated by the (---) in order to show what that would look like in an actual file

Reply:
This could be useful